Makefile

```diff
@@ -13,6 +13,12 @@
 
 all: release
 
+.PHONY: init
+init:
+	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
+	docker build --no-cache -t aws-fluent-bit-initprocess:latest -f Dockerfile.build_init_process .
+	docker build --no-cache -t amazon/aws-for-fluent-bit-init:latest -f Dockerfile.init .
+
 .PHONY: release
 release:
 	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
```

Comment:
add init build in `make release` as well

Reply:
> 

Yeah sure. For the case no env input, this image can run normally like the original image I have already tested. And for wrong env input, the process will be stoped.
And will test more cases and write them in next PR description. Thank you for this advice!

Reply:
Add init build in `make release`, I will try this after figure out the CMD command issue